model/project_configs.go

```diff
@@ -13,10 +13,9 @@ import (
 )
 
 type ProjectConfig struct {
-	Id         string    `yaml:"_id" bson:"_id"`
-	Project    string    `yaml:"project,omitempty" bson:"project,omitempty"`
-	CreateTime time.Time `yaml:"create_time,omitempty" bson:"create_time,omitempty"`
-
+	Id               string    `yaml:"_id" bson:"_id"`
+	Project          string    `yaml:"project,omitempty" bson:"project,omitempty"`
+	ConfigCreateTime time.Time `yaml:"config_create_time,omitempty" bson:"config_create_time,omitempty"`
```

Comment:
What's the purpose for renaming this? (Or having it at all?)

Reply:
It conflicts with the same field that exists in Parser Project during strict validation.